### PR TITLE
Add MP3 banner [WML-5]

### PR DESCRIPTION
**Pull request by pavel-mailpoet**
[WML-5]

[WML-5]: https://mailpoet.atlassian.net/browse/WML-5

### CODE REVIEW

File: src/WPML_OptionsManager.php [hunk 1]
```diff
@@ -536,10 +538,78 @@ public function LogMenu() {
...
+              }
+            ?>
+        </div>
+        <?php
+    }
+
+    private function displayMP3BannerVersionA() {
+        ?>
+            <ul style="list-style-type:disc">
+                <li><?php _e( '50 email templates to chose from', 'wp-mail-logging' );?></li>
```
Comment (wxa) [inline src/WPML_OptionsManager.php]: Typo: chose -> choose.

File: src/WPML_OptionsManager.php [hunk 2]
```diff
@@ -536,10 +538,78 @@ public function LogMenu() {
...
+                </i>
+                — Kida Shey
+            </p>
+        <?php
+    }
+
+    private function displayMP3BannerVersionB() {
+        ?>
+        <p>
+            <?php _e( 'Create beautiful email campaigns and reach your audience in a breeze with the MailPoet  plugin. MailPoet is used by over 300,000 website owners making it the most popular email marketing solution in WordPress. Send beautiful newsletter, notify your readers about last articles and increase your WooCommerce sales!', 'wp-mail-logging' );?>
```
Comment (wxa) [inline src/WPML_OptionsManager.php]: Typo: Send beautiful newsletter -> Send beautiful newsletters.

File: src/WPML_OptionsManager.php [hunk 3]
```diff
@@ -536,10 +538,78 @@ public function LogMenu() {
...
+            <a href="?page=wpml_plugin_log&redirect=free-plan&ref=wml_2" target="_blank">
+                <?php _e( 'Discover MailPoet for free', 'wp-mail-logging' );?>
```
Comment (wxa) [inline src/WPML_OptionsManager.php]: Should this be a blue button?

File: src/WPML_OptionsManager.php [hunk 4]
```diff
@@ -536,10 +538,78 @@ public function LogMenu() {
...
+                    $this->displayMP3BannerVersionB();
+              }
+            ?>
+        </div>
+        <?php
+    }
+
+    private function displayMP3BannerVersionA() {
+        ?>
+            <ul style="list-style-type:disc">
```
Comment (wxa) [inline src/WPML_OptionsManager.php]: Can we align the list bullets with the title?
image

File: src/WPML_OptionsManager.php [hunk 5]
```diff
@@ -536,10 +538,78 @@ public function LogMenu() {
...
+                <li><?php _e( '50 email templates to chose from', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Fun email designer', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Automated email marketing', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'WooCommerce emails', 'wp-mail-logging' );?></li>
+                <li><?php _e( '99.1% success email deliverability', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Fast and friendly support', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Over 100,000 active users', 'wp-mail-logging' );?></li>
```
Comment (wxa) [inline src/WPML_OptionsManager.php]: Should we add semicolons and a comma at the end of the list items?

File: src/WPML_OptionsManager.php (hunk 4 above)
Comment (pavel-mailpoet) [reply src/WPML_OptionsManager.php]: I don't know, this is the default...

File: src/WPML_OptionsManager.php (hunk 3 above)
Comment (pavel-mailpoet) [reply src/WPML_OptionsManager.php]: The spec says link so I am not sure

File: src/WPML_OptionsManager.php (hunk 4 above)
Comment (kgjerstad) [reply src/WPML_OptionsManager.php]: Please indent. Looks ugly like this indeed. I'll let you pick a good value.

File: src/WPML_OptionsManager.php (hunk 5 above)
Comment (kgjerstad) [reply src/WPML_OptionsManager.php]: Nope

File: src/WPML_OptionsManager.php (hunk 3 above)
Comment (kgjerstad) [reply src/WPML_OptionsManager.php]: The main CTA should be a styled as a blue button.

**Comment by wxa** [comment]
Looks good, just the button needs to be in a new paragraph. Fixed.

**Comment by codemonkey-jack** [comment]
@pavel-mailpoet This fails QA due to an issue with the redirect. 

When you click the CTA in the banner:
image

It redirects to: `domain.com/wp-admin/admin.php?page=wpml_plugin_log&redirect=free-plan&ref=wml_2` rather than the MailPoet website as specified in the specs.

**Comment by pavel-mailpoet** [comment]
@codemonkey-jack could you take a look again? I rebased the current master and it should work again.

**Pull request merged**